publisher.go

```diff
@@ -38,12 +38,6 @@ func (p *publisher) Process(s *service, subject string) (result string, err erro
 		result = p.ServicesDeleteError(s)
 	case "service.delete.done":
 		result = p.ServiceDeleteDone(s)
-	case "instances.create":
```

Comment:
As we're using generic publisher for instances, we can remove [InstancesCreate struct](https://github.com/ernestio/workflow-manager/blob/develop/msg.go#L18) and [instances_publisher.go](https://github.com/ernestio/workflow-manager/blob/develop/instances_publisher.go) file.